real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/ComponentData.java

```diff
@@ -0,0 +1,12 @@
+package com.roboclub.robobuggy.ui;
+
+import java.awt.Component;
+
+// a class for storing and passing around the gui component data 
+public class ComponentData{
+	Component component;
+	double percentageLeft;
+	double percentageTop;
+	double percentageWidth;
+	double percentageHeight;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1747252168/issues/source?bid=2938665&fileBranchId=3059924#l11)